release/README.md

```diff
@@ -0,0 +1,22 @@
+# Release tools #
+
+This directory contains tools for Kata Containers releases.
+
+## tag_repos.sh ##
+
+The `tag_repos.sh` script is used to create tags for the Kata Containers
+repositories. This script make sure that all the repositories are in the
+same version (by checking the `VERSION` file).
+
+It will create an **annotated tag** for the new release version for the
+following repositories:
+
+- agent.
```

Comment:
Since the items in this list are not complete sentences I suggest removing the period after each item in lines 14-18.